### PR TITLE
LPS-109009 When portlet.url.anchor.enabled=true, the anchor is added to p_auth parameter in URL

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi @wincent ,

Could you review it, please?

Thanks!

/cc @rgusztus

### CODE REVIEW

**Comment by wincent** [comment]
From: https://github.com/jbalsas/liferay-portal/pull/2151

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js [hunk 1]
```diff
@@ -143,6 +143,10 @@
 
 				var authToken = searchParams.get('p_auth') || '';
 
+				if (authToken.includes('#')) {
+					authToken.substring(0, authToken.indexOf('#'));
+				}
+
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js]: Looks fine to me, presuming that we trust our core-js polyfill to adequately cover `String.prototype.includes` for IE (and I think we do, as we are using `includes` in a bunch of places).

However, problem is, you forgot to reassign back to `authToken`:

```js
authToken = authToken.substring(...);
```

Comment (antonio-ortega) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js]: 🤦‍♂ 

I'll fix it and forward it to Brian. Thanks!

**Comment by antonio-ortega** [comment]
brianchandotcom#85726

**Pull request closed without merging**